interface/src/app/gene/network/network.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.gene.network', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap',
+  'ui.slider'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('gene_network', {
+    url: '/gene_network/?genes&mlmodel',
+    views: {
+      main: {
+        templateUrl: 'gene/network/network.tpl.html',
+        controller: 'GeneNetworkCtrl',
+        controllerAs: 'ctrl'
```

Comment:
I have been thinking about this 'controllerAs' syntax, and it is definitely great that you are using it here, since it is very much considered a best practice. I wonder if we should change the rest of our codebase at some point to use 'controllerAs' syntax instead of '$scope'?